### DISCUSSION TITLE
Шифрование поля (Crypt)

### DISCUSSION THREAD

**Discussion by Sanes** [moonshine-software/moonshine#6] [Q&A] [answered]
В одном из проектов требуется хранение паролей в зашифрованном виде, для последующего отображения.
Сейчас это реализовано так:

При добавлении модели
`$site->password = Crypt::encryptString(Str::random(8));`
Отображение 
`{{ \Crypt::decryptString($site->password) }}`

Как это можно реализовать в MoonShine?

**Comment by lee-to** [comment, marked as answer]
А что мешает использовать простое текстовое поле в moonshine а шифрование и дешифрование вынести в Laravel Accessors/Mutators ?
